build/ghost-buster/Dockerfile

```diff
@@ -1,44 +1,57 @@
-FROM node:0.12
+FROM alpine:latest
```

Comment:
Are you proposing use "latest" or "3.4"? now it is the same, but for future release I'm not 100% sure about back compatibility. Vote for "3.4". Are you?